app/javascript/styles/mastodon/_functions.scss

```
@@ -19,3 +20,11 @@ $lighten-multiplier: 1 !default;
     $space: hsl
   );
 }
+
+@function hex-color($color) {
+  @if type-of($color) == 'color' {
```

Maybe add `@use 'sass:meta';` earlier and use:
```suggestion
  @if meta.type-of($color) == 'color' {
```